internal/fields/validate.go

```diff
@@ -178,6 +178,17 @@ func CreateValidatorForDirectory(fieldsParentDir string, opts ...ValidatorOption
 		return nil, errors.Wrap(err, "can't create field dependency manager")
 	}
 	v.FieldDependencyManager = fdm
+
+	if v.disabledImportAllECSSchema {
+		return v, nil
+	}
+
+	ecsSchema, err := v.FieldDependencyManager.ImportAllFields(defaultExternal)
+	if err != nil {
+		return nil, err
+	}
+	logger.Debug("Adding to the schema the ECS field definitions")
+	v.Schema = append(v.Schema, ecsSchema...)
```

Comment:
I had some issues when creating this type of tests. In tests, it is not found the package root and therefore the dependency management is disabled because of that.

I could add another datastream in the test package where there is a field defined in both in ECS and the package fields.